packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -80,21 +87,29 @@ export const publishInterchainAssetFromBank = async (
       assetKind: AssetKind.NAT,
     },
   };
-  const { creatorFacet: mintP, publicFacet: issuerP } = E.get(
-    E(zoe).startInstance(mintHolder, {}, terms, undefined, keyword),
-  );
 
-  const [issuer, brand] = await Promise.all([issuerP, E(issuerP).getBrand()]);
-  const kit = { mint: mintP, issuer, brand };
+  const startUpgradeable = await startUpgradeableP;
+  produceVBankKits.resolve([]);
+  const { creatorFacet: mint, publicFacet: issuer } = await startUpgradeable({
+    installation: mintHolder,
+    label: keyword,
+    privateArgs: undefined,
```

Comment:
`privateArgs` is optional in StartInstance but I suppose the explicit is helpful for this case.